packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -296,12 +298,40 @@ export const DataGrid: React.FC<GridProps> = (props) => {
     return { onChange: handlePageChange, pageSize: values?.pageSize };
   }, [values?.hidePagination, values?.pageSize]);
 
+  const onSortAction = useEnsembleAction(onSort);
+  // page change action
+  const onSortActionCallback = useCallback(
+    (sorter: Partial<TableProps["onChange"]>) => {
+      if (onSortAction) {
+        onSortAction.callback({ sorter });
```

Comment:
What is `sorter`? Can you add an example of how to use this to make an API call with the new sort values?

Reply:
I added example that how we can use sorter to initiate API call